File: 1. Beginner/1006/1006.scala
object Main {
    def main(args: Array[String]) {
        val a = scala.io.StdIn.readDouble()
        val b = scala.io.StdIn.readDouble()
        val c = scala.io.StdIn.readDouble()
        val r = (a * 2.0 + b * 3.0 + c * 5.0) / 10.0
        println(f"MEDIA = $r%.1f")
    }
}

File: 1006/1006.ts
var inputs = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

var a = Number(inputs.shift());
var b = Number(inputs.shift());
var c = Number(inputs.shift());
   
console.log(`MEDIA = ${(((a * 2 + b * 3 + c * 5) / 10).toFixed(1))}`);